CP-CPS.md

```diff
@@ -0,0 +1,1440 @@
+# 1. INTRODUCTION
+
+## 1.1 Overview
+
+This combined Certificate Policy ("CP") and Certification Practice Statement ("CPS") document outlines the certification services practices for Internet Security Research Group ("ISRG") Public Key Infrastructure ("ISRG PKI").
+
+ISRG PKI services include, but are not limited to, issuing, managing, validating, revoking, and renewing publicly-trusted Certificates in accordance with the requirements of the ISRG Certificate Policy (CP) and in a manner consistent with this CP/CPS.
```

Comment:
Delete "in accordance with the requirements of the ISRG Certificate Policy (CP) and"?

Reply:
We say both "this CP/CPS" and "this document" in various places throughout. Do we want to choose just one wording?

Reply:
> We say both "this CP/CPS" and "this document" in various places throughout. Do we want to choose just one wording?

Let's clean that up after we ship the combined CP/CPS.